test/dbservice/users_test.exs

```diff
@@ -11,7 +11,7 @@ defmodule Dbservice.UsersTest do
     @invalid_attrs %{
       full_name: nil,
       email: nil,
-      phone: nil,
+      phone: "nope",
       gender: nil,
```

Comment:
Also, I think let's remove phone from `validate_required` as there may be some cases where `phone` is null

Reply:
That was already removed